### ISSUE TITLE
Add linter rule for checking rule syntax

### ISSUE THREAD

**Issue by jellizaveta** [AdguardTeam/AGLint#259]
Sometimes we want only some syntax types in our repo, so we need some linter rule to check rules matches with a syntax
e.g.

```
adblock-syntax:
  - preferred:
     - AdGuard
```

this way AGLint should report all non AdGuard rules and should offer conversion as autofix
